benchmarks/transport_montecarlo_main_loop.py

```diff
@@ -0,0 +1,27 @@
+"""
+Basic TARDIS Benchmark.
+"""
+
+from benchmarks.benchmark_base import BenchmarkBase
+from tardis.transport.montecarlo.montecarlo_main_loop import montecarlo_main_loop
+
+
+class BenchmarkTransportMontecarloMainLoop(BenchmarkBase):
+    """
+    class to benchmark montecarlo_main_loop
```

Comment:
Sorry for being too nit-picky, this can be improved.

Reply:
I'm not sure in what sense. Can you please elaborate more?

Reply:
I think Atharva would like a more descriptive docstring